torchci/lib/drciUtils.ts

```diff
@@ -259,9 +259,15 @@ export async function querySimilarFailures(
     baseCommitDate !== "" ? baseCommitDate : job.completed_at
   ).subtract(lookbackPeriodInHours, "hour");
 
+  // Get the workflow name if possible
+  const jobNameIndex = job.name.indexOf(` / ${job.jobName}`);
+  const workflowName =
+    jobNameIndex !== -1 ? job.name.substring(0, jobNameIndex) : "";
```

Comment:
fun fact: if jobNameIndex is negative, then `job.name.substring(0,jobNameIndex)` will always return an empty string anyways.

(But this way is easier to understand)

Reply:
Ah ok, it's good to know. I should spent some time to learn typescript properly instead of picking it up along the way with examples from stackoverflow :P